packages/server/package.json

```diff
@@ -128,10 +133,6 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^14.4.3",
-    "@trpc/client": "^10.0.0-rc.0",
-    "@trpc/next": "^10.0.0-rc.0",
-    "@trpc/react-query": "^10.0.0-rc.0",
-    "@trpc/server": "^10.0.0-rc.0",
```

Comment:
For reference, we want to keep these as devDeps so that our tests can stay as they are.

Without them, everything does "work", except for the tests :/

Reply:
Ref: https://twitter.com/mattpocockuk/status/1586689243702861831